website/www/site/content/en/documentation/io/built-in/hcatalog.md

```diff
@@ -70,7 +70,7 @@ pipeline
 `HCatalogIO` is built for Apache HCatalog versions 2 and up and will not work out of the box for older versions of HCatalog.
 The following illustrates a workaround to work with Hive 1.1.
 
-Include the following Hive 1.2 jars in the über jar you build.
+Include the following Hive 1.2 jars in the uber jar you build.
```

Comment:
This one was correct before, no? I don't know :-)

Reply:
Well, yes, [it comes from a German word](https://stackoverflow.com/a/11947093/10687325) with "ü" as the first character, whenever, iinm, in English we don't have such one. So actually both are correct but I thought that for English version we should stick with just English alphabet.